_usage/limitations-of-babelfish.md

```diff
@@ -302,21 +300,76 @@ features are discovered, and additional features are added to MS SQL:
 | `WAITFOR/RECEIVE` | This syntax is not supported. |
 | `WAITFOR TIME` | This syntax is not supported. |
 | `WITH ENCRYPTION` clause | This syntax is not supported for functions, procedures, triggers, or views. |
-| `CREATE/ALTER/DROP WORKLOAD CLASSIFIER` | Functionality related to this syntax is not supported |
+| `CREATE/ALTER/DROP WORKLOAD CLASSIFIER` | Functionality related to this syntax is not supported. |
 | `CREATE/DROP WORKLOAD CLASSIFIER` | This syntax is not supported. |
 | `WRITETEXT` | This syntax is not supported. |
 | `OPENXML()` | This function is not supported. |
-| XML datatype with schema (xmlschema) | XML type without schema is supported |
-| XML methods | This includes .VALUES(), .NODES(), and other methods |
+| XML datatype with schema (xmlschema) | XML type without schema is supported. |
+| XML methods | This includes .VALUES(), .NODES(), and other methods. |
 | XML indexes | XML indexes are not supported. |
 | XPATH expressions | This syntax is not supported. |
 | WITH XMLNAMESPACES construct | This syntax is not supported. |
 | `WITHOUT SCHEMABINDING` clause | Not supported in functions, procedures, triggers, or views. The object will be created, but as if `WITH SCHEMABINDING` was specified. |
 | `CREATE/ALTER/DROP SELECTIVE XML INDEX` clause | This syntax is not supported. |
 | `CREATE/ALTER/DROP XML SCHEMA COLLECTION` | This syntax is not supported. |
 
-However, more is missing than just features. There are also some
-corner cases which will provide the end user with [different
-behavior](/docs/usage/missing-features) than otherwise expected from MS SQL. It is
-important to understand those corner cases, as well.
+However, more is different than just features.
```

Comment:
correction: "However, there are other differences besides the variations in features."

Reply:
Done.